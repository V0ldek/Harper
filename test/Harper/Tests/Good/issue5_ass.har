fun n = {
  var y = n;
  var n = n;
  n *= 2;
  return y;
};

main = fun 42;