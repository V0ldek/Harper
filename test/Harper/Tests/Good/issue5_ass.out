Execution ended with value: 42